blog/electron-15-0.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Electron 15.0.0
+date: 2021-09-21T00:00:00.000Z
+authors:
+  - name: sofianguy
+    url: 'https://github.com/sofianguy'
+    image_url: 'https://github.com/sofianguy.png?size=96'
+  - name: vertedinde
+    url: 'https://github.com/vertedinde'
+    image_url: 'https://github.com/vertedinde.png?size=96'
+slug: electron-15-0
+---
+
+Electron 15.0.0 has been released! It includes upgrades to Chromium `94`, V8 `9.4`, and Node.js `16.5.0`. We've added several API updates, bug fixes, and general improvements. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 15.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). Continue reading for details about this release and please share any feedback you have!
+
+The Electron team is excited to announce the release of Electron 15.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). Continue reading for details about this release and please share any feedback you have!
+
+## Notable Changes
+
+### Electron Release Cadence Change
+
+Starting with Electron 15, Electron will release a new major stable version every 8 weeks. You can read the [full details here](https://www.electronjs.org/blog/8-week-cadence).
+
+Additionally, Electron will be changing supported versions from latest three versions to latest four versions until May 2022. [See our versioning document](https://electronjs.org/docs/tutorial/electron-versioning)for more detailed information about versioning in Electron.
+
+### Stack Changes
+
+* Chromium `94`
+    * [New in Chrome 94](https://developer.chrome.com/blog/new-in-chrome-94/)
+* Node.js `16.5.0`
+    * [Node 16.5.0 blog post](https://nodejs.org/en/blog/release/v16.5.0/)
+* V8 `9.4`
+    * [V8 9.4 blog post](https://v8.dev/blog/v8-release-94)
+
+### Highlight Features
+
+* 
+
+See the [15.0.0 release notes](https://github.com/electron/electron/releases/tag/v15.0.0) for a full list of new features and changes.
+
+## Breaking Changes
+
+### Default Changed: nativeWindowOpen defaults to true
+Prior to Electron 15, window.open was by default shimmed to use BrowserWindowProxy. This meant that window.open('about:blank') did not work to open synchronously scriptable child windows, among other incompatibilities. nativeWindowOpen: true is no longer experimental, and is now the default.
+
+See the documentation for window.open in Electron for more details.
+
+More information about future changes can be found on the [Planned Breaking Changes](https://www.electronjs.org/docs/breaking-changes) page.
```

Comment:
```suggestion

```
Moving this line up so it doesn't appear to be part of the nativeWindowOpen header